### PR TITLE
Create player question page

### PR DESCRIPTION
**Pull request by sebcontreras**
In this PR:
- Created question page
- Created reusable question component
- Created reusable answer component

Notes:
- I hardcoded the text elements for now so that the team can have a look. Once the backend is hooked up I can change these hardcoded values into elements from the state store
- I merged Vic's start page branch into this one so that I could use the router that he setup. So just make sure to: 
          - Merge his PR before this one.
          - Ignore the changes that he made when looking at this PR

### CODE REVIEW

**Review by vicphan — COMMENTED** [review]
image
Overall good work. Don't see any necessary changes on desktop view but just some minor suggestions for the mobile view:
* Maybe add some margin to the cards so there's a little space between the edge of the screen and the cards?
* And maybe slightly increase the font size? It's a little small on mobile

File: src/components/Answer.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { Button } from '@chakra-ui/react';
+
+export default function Answer(props: {background: string, text: string}){
```
Comment (vicphan) [inline src/components/Answer.tsx:3]: Might be a good idea to assign some sort of id for answer. Would be useful when processing (i.e. sending it over to the backend)

File: src/components/Question.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Card, CardHeader, Heading, CardBody, Text } from '@chakra-ui/react'
+
+export default function Question(props: {title: string, text: string}){
```
Comment (vicphan) [inline src/components/Question.tsx:3]: Same suggestion as answer (add id)

File: src/components/Answer.tsx (hunk 1 above)
Comment (sebcontreras) [reply src/components/Answer.tsx:3]: Good call, we'll definitely need this at some point

**Pull request merged**